doc/developer/build_release.rst.tmpl

```diff
@@ -19,175 +19,165 @@ the first bug fix release, and ``<version>`` can be ``2.0``, ``2.1``, ...
+On ``ngeo`` you should create the new version branch.
 
-c2cgeoportal
-------------
+On the new version branch, in the file ``.github/workflows/ci.yaml`` set ``MAIN_BRANCH`` to
+``<new version>``.
 
-New version
-~~~~~~~~~~~
+On the master branch, in the ``Makefile`` you should set ``DEMO_BRANCH`` to ``prod-<next version>``.
 
-Checkout the code:
+On the master branch, in the file ``package.json`` bump the package version.
 
-.. prompt:: bash
+On ``c2cgeoportal`` you should create the new version branch.
 
-    git fetch
-    git checkout master
-    git reset --hard origin/master
+On the new version branch, in the file ``.github/workflows/ci.yaml`` set ``MAIN_BRANCH`` to
+``<new version>``.
 
-Create a new Transifex resource:
-
-.. prompt:: bash
+On the master branch, in the file ``.github/workflows/ci.yaml`` set ``MAJOR_VERSION`` to
+``<next version>``.
 
-    make build-tools
-    docker run --user=root camptocamp/geomapfish-tools transifex-init \
-        <version> <version + 1> \
-        "$(gopass gs/ci/transifex/rw/username)" \
-        "$(gopass gs/ci/transifex/rw/password)"
+Create the new Transifex ressources
+-----------------------------------
 
-For each version, we create a new branch (at the latest, for the final release):
+On ngeo checkout the new version branch
 
 .. prompt:: bash
 
-    git checkout -b <version>
-    git push origin <version>
+   make transifex-get
 
-Change the version in the following files:
- * ``.github/workflows/ci.yaml`` (``MAIN_BRANCH``, ``MAJOR_VERSION``)
 
+Checkout the master branch
 
-Commit your changes:
+In the ``Makefile`` set ``TX_VERSION`` to ``<next version>``.
 
 .. prompt:: bash
 
-    git add .github/workflows/ci.yaml
-    git commit -m "Create the version <version> branch"
+   rm .tx/config contribs/gmf/apps/.tx/config
+   make transifex-init
 
-Go back to the master branch:
+On ``c2cgeoportal``, new version branch do:
 
 .. prompt:: bash
 
-    git checkout master
-
-Copy ``.github/workflows/ci.yaml`` to ``.github/workflows/rebuild-<version>.yaml``
-
-Change the on from pull to cron and checkout the right branch:
-
-.. code:: diff
-
-     on:
-    -  push:
-    +  schedule:
-    +    - cron: "30 2 * * *"
-
-     - uses: actions/checkout@v1
-    +  with:
-    +    ref: <version>
-
-Commit your changes:
-
-.. prompt:: bash
+    make build-tools
+    docker run --user=root camptocamp/geomapfish-tools transifex-init \
+        <version> <version + 1> \
+        "$(gopass gs/ci/transifex/rw/username)" \
+        "$(gopass gs/ci/transifex/rw/password)"
 
-    git add .github/workflows/ci.yaml
-    git commit -m "Start version <version + 1>"
+Configure dependabot
+--------------------
 
-Push your changes:
+On ``c2cgeoporal`` and ``ngeo`` master branches, configure the updates for the security issues, see also:
+`c2cgeoportal <https://github.com/camptocamp/c2cgeoportal/commit/7783a2d1d02cc58da7cb1e3b9d32a1358cc30524>`_
```

Comment:
It is not clear to me why we are referencing this commit

Reply:
To give an example on how to configure Dependabot for the new branch...

Reply:
How about:
..., add in the dependabot configuration the new GMF version to receive updates for security issues; see for example this commit:

Reply:
Or:
On ``c2cgeoportal`` and ``ngeo`` master branches, add in the Dependabot configuration the new branch to
receive updates for security issues; see for example those commits: